docs/getting_started.rst

```diff
@@ -394,6 +407,10 @@ and ``couchdb_admin_party`` context helpers.
 
         print('Databases: {0}'.format(client.all_dbs()))
 
+.. warning:: Don't use the ``doc.delete()`` method inside your ``Document``
+             context manager. This method immediately deletes the document on
+             the server.
```

Comment:
>Okay, let's advise deleting inside the CM is done via doc['_deleted'] = True. That seems to work well.

Cool, perhaps we should advise wiping the non `_id` `_rev` `_deleted` keys too or at least mention that for cases where people might not want to send a large body back again?

>doc.delete() works inside the CM (assuming the doc already exists). Although on exiting the CM we always write a doc too. So using .delete() in the CM doesn't really make any sense.

I thought there was a DB CM as well as a doc CM? In any case I'm happy to go with `_deleted` approach.

>Noting also that using the Document CM doesn't update the local cache.

We probably need a clear document about what exactly happens local vs remote. I guess that's just another part of #116.